debian/changelog

```diff
@@ -1,3 +1,60 @@
+ubuntu-advantage-tools (27.1) impish; urgency=medium
+
+  * d/control:
+    - specify debianutils min version
+  * lintian:
+    - override xenial specific errors
+    - rename package-specific overrides for pro vs tools
+    - override ubuntu-advantage-pro wanted-by-target cloud-init
+  * New upstream release 27.1:
+    - changelog: fix lintian typos amend and redact incorrect 27.0 entry
+      (GH: #1624)
+    - apt-hook:
```

Comment:
I think we can sort the topics and subtopics in the changelog. However, let's keep the tests and jenkins change in the end